Sources/Stats.php

```diff
@@ -779,7 +779,7 @@ function SMStats()
 		'topics' => $modSettings['totalTopics'],
 		'boards' => 0,
 		'php_version' => $serverVersions['php']['version'],
-		'database_type' => strtolower($serverVersions['db_server']['title']),
+		'database_type' => strtolower($serverVersions['db_engine']['version']),
```

Comment:
With this you get the engine but not any more the database version.
In my eyes is version more importent as the engine.
Engine with version would be fine too.
But maybe you should splitt this in two information,
database_type (with the version nummer for the sake of backward compatibility)
database_vendor to know which product.